homeassistant/components/zwave/__init__.py

```diff
@@ -752,6 +774,9 @@ def _finalize_start():
 
     hass.bus.async_listen_once(EVENT_HOMEASSISTANT_START, start_zwave)
 
+    hass.services.async_register(DOMAIN, const.SERVICE_START_NETWORK,
```

Comment:
Why did you move this service?

Reply:
When is this service even useful ? Don't we always start the network on startup ?

Reply:
It should always start on start-up, but if anything prevents it from starting then you're stuck restarting HA again instead of just hitting the Start Network button in the Z-Wave config panel.